lib/source-destination/progress.ts

```diff
@@ -54,10 +54,12 @@ export function makeClassEmitProgressEvents<
 				averageSpeed: 0,
 			};
 
+			// eslint-disable-next-line
```

Comment:
> balena-lint expects us to replace `@ts-ignore`s with `@ts-expect-error`s with a description, not silence them.
Even if the description is `to make TS happy/ I don't know why` it's way more helpful, since if we later fix the typings, TS will let us know that the `@ts-expect-error` is no longer required.

Reply:
makes total sense, will switch everywhere